src/IO/lbsStatistics.py

```diff
@@ -107,6 +107,42 @@ def sampler(distribution_name, parameters, logger: Logger = None):
         logger.error(f"{distribution_name} distribution is not supported.")
         return None, None
 
+def Hamming_distance(arrangement_1, arrangement_2):
+    """Compute Hamming distance between two arrangements
+    """
+
+    # Distance can only be compute between same length arrangements
+    if len(arrangement_1) != len(arrangement_2):
+        return math.inf
+
+    # Iterate over arrangement values
+    hd = 0
+    for i, j in zip(arrangement_1, arrangement_2):
+        # Increment distance for each pair of different entries
+        if i != j:
+            hd += 1
+
+    # Return the final count of differences
+    return hd
+
+
+def min_Hamming_distance(arrangement, arrangement_list):
```

Comment:
function name `Hamming_distance` should be lowercase [PEP8 function and variable names](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)

Reply:
But "**H**amming" is a proper noun (it's not _h_amming). Should we still lowercase it?

Reply:
> But "**H**amming" is a proper noun (it's not _h_ amming). Should we still lowercase it?

it should be called `hamming_distance`

Reply:
Yes but as I said "Hamming" is a **proper noun**. It's the last name of a [Richard Hamming](https://en.wikipedia.org/wiki/Richard_Hamming)
It is improper English syntax to lowercase the first letter of proper nouns. Are you **certain** that the PEP standard is requiring this for proper nouns as well?

Reply:
From [wiki](https://en.wikipedia.org/wiki/Hamming_distance) :)
![Zrzut ekranu z 2022-02-21 19-54-10](https://user-images.githubusercontent.com/47597569/155012510-3ff052b4-a12f-4792-94bf-c76a712aff57.png)